README.md

```diff
@@ -473,6 +473,37 @@ You might also want to back-up your master key, but do NOT store it in
 the repository - put it safely away in e.g. your password manager's
 secure notes -section.
 
+You can export the master key by running:
+
+```bash
+poetry run invoke get-master-key --env <env>
+```
+
+This will save the key as `envs/<env>/master.key`. Make sure to NOT under any circumstance commit this file.
+
+#### Working with SealedSecrets
+
+If you have the `master.key` file for an environment (or sufficient privileges to retrieve it from the cluster),
+you can use the following command to unseal the secrets to a readable and editable format:
+
+```bash
+poetry run invoke unseal-secrets --env <env>
+```
+This will convert each of the files in `envs/<env>/secrets/*.yaml` to a corresponding
+`*.unsealed-secrets.yaml` file. This will also base64 decode the file, so you can see and edit the actual values easily.
+
+N.B. Make sure to not under any circumstance commit the `*.unsealed-secrets.yaml` or `master.key` files!
```

Comment:
`N.B.` is not a very widely known notation :) How about just `NOTE:`

Reply:
Changed according to suggestion.